src/collectd_cvmfs/__init__.py

```diff
@@ -54,9 +72,10 @@ def read(self, config):
 
             if config.mounttime:
```

Comment:
Does it make sense "mounttime"? We always have to mount the repo if we wan to get metrics.
We could:
* Drop the parameter.
* Keep the parameter and always mount but not send the metric if not true.

Reply:
Can you alarm on absence or freshness of a metric?